src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/OfflineAuthenticationTest.java

```diff
@@ -174,38 +178,16 @@ void GIVEN_clientDevice_WHEN_verifyingItsIdentity_THEN_pemStored() throws Interr
    void GIVEN_storedCertificates_WHEN_refreshEnabled_THEN_storedCertificatesRefreshed() throws
            NoSuchAlgorithmException, CertificateException, OperatorCreationException, IOException,
            InvalidCertificateException, InterruptedException {
-       // Given
```

Comment:
Extracted all these into methods so I could reuse it and remove clutter from the test definition